docs/worker-node/install-singularity.md

```diff
@@ -59,22 +59,17 @@ There are two sets of instructions on this page:
 - [Enabling Unprivileged Singularity](#enabling-unprivileged-singularity)
 - [Singularity via RPM](#singularity-via-rpm)
 
-OSG VOs are working to support running singularity directly from OASIS,
-the OSG Software [CVMFS distribution](install-cvmfs), when unprivileged
-singularity is enabled.  At that point sites will not have to install
-the singularity RPM themselves.  As of April 2019, no VO in the OSG is
-yet ready to do this, but OSG recommends that all RHEL 7.x installations
-enable support for unprivileged singularity and for now also install the
-RPM.  Sites may also choose to configure their RHEL 7.x RPM
-installations to run unprivileged.  RHEL 6.x installations have no
-option for unprivileged singularity so there the RPM has to be installed
-and left configured as privileged.
+OSG VOs all support running singularity directly from CVMFS when unprivileged
+singularity is enabled.  OSG recommends that all RHEL 7.x installations
+enable support for unprivileged singularity instead of installing the
```

Comment:
```suggestion
enable support for unprivileged singularity via OASIS instead of installing the
```

The way this sentence is currently worded makes it sound like that unpriv singularity and RPMs are mutually exclusive

Reply:
That's what I intended it to say.  I want to recommend that people enable unprivileged singularity and not install rpms.  The OSG security draft I wrote encourages people to remove the singularity rpms if they have no non-OSG requirement for it.

Reply:
Yeah, I think that recommendation is fine and my addition only clarifies that. The existing sentence sounds like you can't have unprivileged Singularity with an RPM, which is confusing with the following sentence that says the RPM can be configured to run unprivileged.

Reply:
Oh, right.  I would at minimum use CVMFS instead of OASIS, since the previous sentence now says CVMFS (a recent change because ATLAS doesn't use singularity from the oasis repository).  Even so, I don't like the change, because enabling unprivileged singularity doesn't only enable it to be run from CVMFS.  Enabling unprivileged singularity is exactly what we want them to do instead of installing the rpm.  How about if it said "OSG recommends that all RHEL 7.x worker nodes have unprivileged singularity enabled and not have the RPM installed when possible."  Or "OSG recommends that system administrators enable unprivileged singularity on RHEL 7.x worker nodes and not install the singularity RPM when possible."

Reply:
>  Or "OSG recommends that system administrators enable unprivileged singularity on RHEL 7.x worker nodes and not install the singularity RPM when possible."

I like this one!

Reply:
done